pandas/core/generic.py

```diff
@@ -3455,7 +3455,7 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
         ----------
         method : {'linear', 'time', 'index', 'values', 'nearest', 'zero',
                   'slinear', 'quadratic', 'cubic', 'barycentric', 'krogh',
-                  'polynomial', 'spline' 'piecewise_polynomial', 'pchip',
+                  'polynomial', 'spline' 'from_derivatives', 'pchip',
```

Comment:
So we need a check near the top of `interpolate` in `pandas.core.generic` as well right?